docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+
+  thrift:
+    build: docker/thrift
+    ports:
+      - "10000:10000"
+      - "4040:4040"
+    depends_on:
+      - hive-metastore
+    volumes:
+      - ./.spark-warehouse/:/usr/local/spark/spark-warehouse
```

Comment:
Just curious, when would you use this? The data will be mounted onto the root fs, while all the metadata is inside the docker images.

Reply:
I mount another volume here: https://github.com/fishtown-analytics/dbt-spark/pull/58/files#diff-4e5e90c6228fd48698d074241c2ba760R20

So, you have both the metadata and data persisted locally

I just think it's nicer to know you can run `docker-compose down` in case something is wrong but still keep you metadata/data somewhere